tests/integration/targets/account_availability_info/tasks/main.yaml

```diff
@@ -9,6 +9,7 @@
       assert:
         that:
           - account_availability.account_availability.region == "us-east"
+          - account_availability.account_availability.available | length + account_availability.account_availability.unavailable | length == 4
```

Comment:
Would it worth it to also check for the actual content? e.g. ["Linodes", "NodeBalancers", "Block Storage", "Kubernetes"]

Reply:
I don't think we can guarantee the length of the availabilities either, so it might be make sense to check if certain entries exist in the fields rather than explicitly checking the list as a whole :+1:

Reply:
Good point! The content of available and unavailable may change due to data center capability, so it may break our test if we check the content. However, the "total" will always be the four services: Linodes, NodeBalancers, Block Storage, Kubernetes. At least this won't change based on the current design ;) I've added an comment to describe it.